Packages/com.unity.inputsystem/Documentation~/HID.md

```diff
@@ -21,19 +21,108 @@ When the Input System automatically creates a layout for an HID, it always repor
 
 The auto-generated layouts represent a "best effort" on the part of the Input System. The way Human Interface Devices describe themselves in accordance with the HID standard is too ambiguous in practice, so generated layouts might lead to Controls that don't work as expected. For example, while the layout builder can identify hat switches and D-pads, it can often only make guesses as to which direction represents which. The same goes for individual buttons, which generally aren't assigned any meaning in HID.
 
-The best way to resolve the situation of HIDs not working as expected is to add a custom layout, which bypasses auto-generation altogether. See [Overriding the HID fallback](#overriding-the-hid-fallback) for details.
+The best way to resolve the situation of HIDs not working as expected is to add a custom layout, which bypasses auto-generation altogether. See [Overriding the HID fallback](#creating-a-custom-device-layout) for details.
 
 ## HID output
 
 HIDs can support output (for example, to toggle lights or force feedback motors on a gamepad). Unity controls output by sending HID Output Report commands to a Device. Output reports use Device-specific data formats. To use HID Output Reports, call [`InputDevice.ExecuteCommand`](../api/UnityEngine.InputSystem.InputDevice.html#UnityEngine_InputSystem_InputDevice_ExecuteCommand__1___0__) to send a command struct with the [`typeStatic`](../api/UnityEngine.InputSystem.LowLevel.IInputDeviceCommandInfo.html#properties) property set as `"HIDO"` to a Device. The command struct contains the Device-specific data sent out to the HID.
 
-## Overriding the HID fallback
+## Creating a custom device layout
 
 Often, when using the layouts auto-generated for HIDs, the result isn't ideal. Controls don't receive proper names specific to the Device, some Controls might not work as expected, and some Controls that use vendor-specific formats might not appear altogether.
 
-The best way to deal with this is to set up a custom Device layout specifically for your Device. This overrides the default auto-generation and gives you full control over how the Device is exposed.
+The best way to deal with this is to override the HID fallback and set up a custom Device layout specifically for your Device. This overrides the default auto-generation and gives you full control over how the Device is exposed.
 
-### Example workflow
+Below are three example workflows showing different ways to achieve this.
+
+- [Example 1 - Use an existing C# InputDevice](#custom-device-workflow-example-1---use-an-existing-c-inputdevice)
+- [Example 2 - Create your own InputDevice class](#custom-device-workflow-example-2---create-your-own-inputdevice-class)
+- [Example 3 - A more complex example using the PS4 DualShock Controller](#custom-device-workflow-example-3---ps4-dualshock-controller)
+
+### Custom Device Workflow Example 1 - Use an existing C# InputDevice
```

Comment:
Some of this seems to be duplicated in https://docs.unity3d.com/Packages/com.unity.inputsystem@1.5/manual/Layouts.html

I wonder if we should/could remove some of this duplication

Reply:
We probably should, however this duplication must have already been present in the old How Do I page because I moved the information from there, so this is not a regression as such. If all the other errors are resolved, I'd like to get this PR merged in and I'll create a separate ticket to examine the de-duplication of material between this page and the layouts page.